lib/stdlib/test/zip_SUITE.erl

```diff
@@ -1558,6 +1558,37 @@ extended_timestamp(Config) ->
 
     ok.
 
+% checks that the timestamps in the zip file are capped to 59 seconds
+capped_timestamp(Config) ->
+
+    DataDir = get_value(data_dir, Config),
+    Archive = filename:join(DataDir, "bad_seconds.zip"),
+    PrivDir =  get_value(pdir, Config),
+    ExtractDir = filename:join(PrivDir, "extract"),
+
+    {ok, [#zip_comment{},
+          #zip_file{ info = ZipFI = #file_info{ mtime = ZMtime}} ]} =
+        zip:list_dir(Archive),
+
+    ct:log("in zip : ~p",[ZipFI]),
+    ct:log("zipinfo:~n~ts",[os:cmd("zipinfo -v "++Archive)]),
+
+    % and not {{2024, 12, 31}, {23, 59, 60}}
+    ?assertEqual({{2024, 12, 31}, {23, 59, 59}}, ZMtime),
+
+    ok = file:make_dir(ExtractDir),
+    ?assertMatch(
+       {ok, ["testfile.txt"]},
+       unzip(Config, Archive, [{cwd,ExtractDir}])),
+
+    {ok, UnzipFI } =
+        file:read_file_info(filename:join(ExtractDir, "testfile.txt"),[raw]),
+
+    ct:log("extract: ~p",[UnzipFI]),
+    % UnzipMode = un_z64(get_value(unzip, Config)),
+    % assert_timestamp(UnzipMode, UnzipFI, ZMtime),
```

Comment:
I'm commenting this one out for now since it's failing and I am not sure what to do about it :)